### PR TITLE
Site update 2019

### PR DESCRIPTION
**Pull request by alicetragedy**
Update the site to reflect changes to the 2019 format and communication around our announcement; specifically updating the timeline, the guides, our call to action on the landing page and removing unnecessary content and links in the navigation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,7 +82,12 @@ <h2>Become a sponsor</h2>
     <div class="col-md-6">
       <h3>Timeline 2019</h3>
       <div class="squiggly-line"></div>
-      <p>To be announced!</p>
+      <ul class="list list--none">
+        <li><span class="fa fa-star color-red"></span> 21 Mar–22 Apr: Call for projects</li>
```
Comment (alicetragedy) [inline index.html:86]: note to self: until Apr 29?

File: index.html [hunk 2]
```diff
@@ -82,7 +82,12 @@ <h2>Become a sponsor</h2>
...
+        <li><span class="fa fa-star color-red"></span> 21 Mar: Rebranding starts</li>
+        <li><span class="fa fa-star color-red"></span> 29 Apr: Publish list of projects</li>
```
Comment (alicetragedy) [inline index.html:88]: update to May 6th?

File: guide/projects.md [hunk 3]
```diff
@@ -82,16 +80,16 @@ Furthermore, you will get:
 * All in all: the chance to develop a sustainable Open Source project
 
 
-
 <h3 id="q5">How do I submit my project?</h3>
 
-Project mentors can submit their projects using our [Teams App](https://teams.railsgirlssummerofcode.org). The deadline for Rails Girls Summer of Code 2018 is January 28th, 2018 (23:59:00 UTC).
+Project mentors can submit their projects using our [Teams App](https://teams.railsgirlssummerofcode.org). We've set the deadline for this year's program to April 22nd, 2019 (23:59:00 UTC) to give us some time to look at the submissions and publish a comprehensive list of projects.
```
Comment (alicetragedy) [inline guide/projects.md:85]: April 29th?

**Review by nynnejc — APPROVED** [review]
Looks good, glad to see the info out

**Pull request merged**